src/cmd/alias.go

```diff
@@ -22,6 +22,12 @@ var commandsToAlias = []string{
 var aliasCommand = &cobra.Command{
 	Use:   "alias (true | false)",
 	Short: "Adds or removes default global aliases",
+	Long: `Adds or removes default global aliases
```

Comment:
Do you think its worth it to try and DRY up the repetition of the short form inside the long form with a variable? Seems to be a pattern on the all the commands